packages/web-test-helpers/src/mocks/defaultComponentMocks.ts

```diff
@@ -9,18 +9,29 @@ import {
 } from '../../../web-pkg/src/'
 import { Router, RouteLocationNormalizedLoaded, RouteLocationRaw } from 'vue-router'
 import { ref } from 'vue'
+import { writable } from '../helpers'
 
 export interface ComponentMocksOptions {
   currentRoute?: RouteLocationNormalizedLoaded
 }
 
 export const defaultComponentMocks = ({ currentRoute = undefined }: ComponentMocksOptions = {}) => {
-  const $router = mockDeep<Router>({ ...(currentRoute && { currentRoute: ref(currentRoute) }) })
+  const $router = mock<Router>()
+  if (currentRoute) {
+    writable($router).currentRoute = ref(currentRoute)
+  } else {
+    writable($router).currentRoute = ref(
+      mock<RouteLocationNormalizedLoaded>({ name: '', path: '' })
+    )
+  }
```

Comment:
Again, not sure why this is needed. What I got out here: 

* We should never use `deepMock` because it often causes trouble. This has been the case in the past as well.
* As described above, setting values via `mock.whatever = ...` might be a cleaner approach rather than initializing the mock with default values.